### PR TITLE
[Feature] Add staff page

### PR DESCRIPTION
**Pull request by JeremySungTw**
## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
新增 2020 工作人員清單頁面

## Steps to Test This Pull Request
1. 訪問 `/en-us/about/staff/` 和 `/zh-hant/about/staff/` 頁面
2. 確認組別順序與工作人員姓名

## Related Issue
#849 

## More Information

待各組提供工作人員姓名

**Screenshots**
### ZH
image
### EN
image

### CODE REVIEW

File: src/locale/en_US/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2820,7 +2820,13 @@ msgctxt "staff"
...
 msgstr "Venue"
 
-#: templates/pycontw-2018/_includes/staff.html:98
+#: templates/pycontw-2018/_includes/staff.html:97
+#: templates/pycontw-2019/_includes/staff.html:104
+msgctxt "staff"
+msgid "Recruitment"
+msgstr "Recruitment"
+
+#: templates/pycontw-2018/_includes/staff.html:107
```
Comment (mattwang44) [inline src/locale/en_US/LC_MESSAGES/django.po:2829]: The line number seems incorrect. Please check again. 
Also, remember to add `#: templates/pycontw-2020/_includes/staff.html:XXX` to all the terms after all the staff lists are collected.
```suggestion
#: templates/pycontw-2020/_includes/staff.html:XXX
msgctxt "staff"
msgid "Recruitment"
msgstr "Recruitment"

#: templates/pycontw-2018/_includes/staff.html:98
```

Comment (JeremySungTw) [reply src/locale/en_US/LC_MESSAGES/django.po:2829]: 好的，我會在人員清單完成後檢查行數。

File: src/templates/pycontw-2020/_includes/staff.html [hunk 2]
```diff
@@ -12,13 +12,28 @@ <h2>{% trans 'Registration' context 'staff' %}</h2>
...
+	<li>Macs</li>
+	<li>Maliao Guo</li>
+	<li>Annie Bu</li>
+	<li>Pearlie</li>
+	<li>GTB</li>
+	<li>Jeremy Sung</li>
+	<li>Leon Fu</li>
+	<li>Josix</li>
+	<li>SHY</li>
+	<li>Fliss</li>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/_includes/staff.html:30]: He recently changed his name to `Flynn`. Please help to update this. thx!
```suggestion
	<li>Flynn</li>
```

**Comment by yychen** [comment]
那個 XD branch name 不要用 # 符號吧(?)

**Comment by yychen** [comment]
註冊組還沒填寫的話，要不要先把它拿掉，等到資料拿到了再放呢? 擺著空白有點怪...

File: src/templates/pycontw-2020/contents/_staff.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{% extends 'contents/_base.html' %}
+
+{% load i18n %}
+
+{% block title %}{% trans 'Staff' %}{% endblock title %}
+
+{% block content %}
+
+<h1>{% trans 'Staff' %}</h1>
```
Comment (mattwang44) [inline src/templates/pycontw-2020/contents/_staff.html:9]: Please update the style based on #882

**Pull request merged**